src/index.html

```diff
@@ -13,6 +13,7 @@
     />
 
     <link rel="stylesheet" href="output.css" />
+    <link rel="stylesheet" href="main.css">
```

Comment:
Please let us avoid the use of custom CSS. I suggest you remove this line.

Reply:
Alright sure.